docs/reference/server-configuration.md

```diff
@@ -346,7 +412,7 @@ interactive mode. For more information, see
 
 ## `ksql.persistence.default.format.key`
 
-**Per query:** yes
+**Per query:** yes (for {{ site.ccloud }}, settable with ALTER SYSTEM only)
```

Comment:
Actually I think we missed these configs as well, they should be per-query and thus settable via `SET`. You have this backwards here, `ALTER SYSTEM` is for system-level configs which are not per-query. (But as noted in this comment, not all configs which are not per-query can be changed via `ALTER SYSTEM`

Of those not-per-query configs, pretty much those with the prefix `ksql.{streams/consumer/producer}` on this are system properties and settable via `ALTER SYSTEM`, everything else is a system config which can NOT be changed via `ALTER SYSTEM` (I know this is confusing because we just changed this)

Reply:
@rodesai sounds like this one also does not make sense as a per-query config, agreed?

```suggestion
**Per query:** no
```